app/src/main/java/com/google/android/stardroid/search/PrefixStore.java

```diff
@@ -14,11 +14,7 @@
 
 package com.google.android.stardroid.search;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
```

Comment:
Done.

Maybe a link in README to: https://google.github.io/styleguide/javaguide.html?